.github/workflows/generate-modified-test-wrappers.yaml

```diff
@@ -82,8 +87,13 @@ jobs:
         if: ${{ inputs.wasm-path }}
         run: echo "POLYWRAP_WASM_PATH=${{ github.workspace }}/${{ inputs.wasm-path }}" >> $GITHUB_ENV
 
-      - name: Run wrappers build
-        run: cargo run -- -w
+      - uses: nick-fields/retry@v2
```

Comment:
Let's move this bug fix into a separate PR? This fix seems like a non-controversial one, but the other workflow changes in `ci-javascript` and `ci-rust` are a bit more radicle.